test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -9199,110 +9199,6 @@ TEST(mongoUpdateContextRequest, servicePathEntityUpdate_3levels)
 
 /* ****************************************************************************
 *
-* servicePathEntityUpdates_2levels -
-*
-*/
-TEST(mongoUpdateContextRequest, servicePathEntityUpdate_2levels)
```

Comment:
They way I saw it they test that the broker works the way it should NOT work - the bug fixed in this PR.
However, you are definitely better than I am 'dechiffering' unit tests.
If you don't have the same feeling that I had then something might be wrong ...

So, please look again and make sure that the three tests are OK (and that they do NOT test the invalid way of working that was just fixed in this PR).